### DISCUSSION TITLE
Contribution graph not showing some of my commits - breaks streaks

### DISCUSSION THREAD

**Discussion by realitymolder** [community/community#33023] [Other Feature Feedback, Questions, & Ideas]
Hey, It seems like there is an issue with the contribution graph. 
I have contributed for a while and two times already some of my streak has been stopped by "removing" some of my commits from the graph. I'm not quite sure why.

1. I took in consideration the fact that only commits to the main branch counted.
2. I took in consideration the fact that my email should be the same for the commit. 

From Aug technically everything should be filled 
![image](https://user-images.githubusercontent.com/26459151/190116637-df614059-56cb-4555-940e-58d9279bdd1a.png)

**Comment by radar-blip** [comment]
Here's some info on [Contributions graph](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile) that might help.

**Reply by realitymolder** [reply]
Hey, I was reading this before posting my question. 
I'm mainly referring to the fact that my contributions were recorded already and somehow gone.

**Reply by radar-blip** [reply]
Did they come back?

**Reply by realitymolder** [reply]
> Did they come back?

sadly no.

**Reply by radar-blip** [reply]
If you changed the visibility of the repo the commits were made on (made it private) the contributions would then be hidden.
Thats the only other thing I can thing of.

See  Number 2 here: [Private Contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile)

**Comment by jfullstackdev** [comment]
one, you have unverified commits

![image](https://user-images.githubusercontent.com/47092464/190640047-ad7fd6c2-5862-4ec2-882f-8bb5891f5ad3.png)

two, it's already updated I think so, but maybe what you are seeing is a cached data, try logging in in Incognito mode

this is what I'm seeing on my end 

![image](https://user-images.githubusercontent.com/47092464/190639957-f48dad62-1709-4ec4-b7f2-d78815fbd97c.png)

**Reply by realitymolder** [reply]
Thanks, Since then I've updated my GPG key. That said, it seems like the commits are still missing, and I'm not sure why. 
I'm referring to commits from The end of July till current day. Basically, there should be a streak.

**Reply by jfullstackdev** [reply]
if you can identify those commits like, if say, it's another repo, try checking those commits against the commits you have at your profile or make sure it's either verified or not or the repo is not a fork because it's still not counted if that is on the main branch of a fork

**Comment by babihutan** [comment]
I have this problem.  I see the commits on master in github, but they do not count towards my contributions and breaks my streak.  I am suspecting github did something and for some period of time, something got broken.  Nothing changed on my end, and today I was able to post a commit that was posted successfully.  Is there a way to regen the commit counts?

**Comment by ashimdahal** [comment]
I've had this for Jan 1; the contribution shows on the repo as Jan 1 but on the contribution graph it shows up as Jan 2

![image](https://github.com/user-attachments/assets/a30418bc-dcb5-43c1-a9db-d61c319ada10)
![image](https://github.com/user-attachments/assets/df453ff5-9b61-4db8-b5ba-fbde750042e5)
